source/manual/alerts/email-alert-api-unprocessed-content-changes.html.md

```diff
@@ -63,11 +64,23 @@ ContentChange.where("created_at < ?", 10.minutes.ago).where(processed_at: nil).m
 
 - [content_change_failed_emails][]
 
+#### Resend failed emails by email ids
+
+- [resend_failed_emails:by_id][]
+
+#### Resend failed emails by date range
+
+- [resend_failed_emails:by_date][]
+
```

Comment:
Thanks for putting this together @laurentqro, after chatting with Ben earlier about whether rake task is a good idea or not he raised that these docs are at risk of becoming a bit of a task buffet with various code snippets that slip out of date and I was inclined to agree.

In light of this I wonder if we can cut the information down further so there is less to maintain and more focused. I've had a quick bash at reworking your words in this way, let me know if you think it'd be a good approach.

```
This alert is triggered when content changes are not processed within the time we would expect.

* `warning` - `content_changes` unprocessed for over 90 minutes
* `critical` - `content_changes` unprocessed for over 120 minutes

Every 30 minutes, [RecoverLostJobsWorker][recover-lost-jobs-worker] automatically requeues jobs for any
unprocessed content changes and messages that are over 1-hour old.

If you see this alert it is likely that something has broken in Email Alert API that is either blocking one or all recovery attempts.

To diagnose this problem you should consult [Sentry](https://sentry.io/organizations/govuk/issues/?project=202220) to see if there are errors reported. If you find nothing conclusive in Sentry you should check [Email Alert API sidekiq logs](https://docs.publishing.service.gov.uk/manual/logging.html#kibana) to check the jobs are running correctly.

If these don't yield any results you can try running the [RecoverLostJobsWorker][recover-lost-jobs-worker] and/or [ProcessContentChangeWorker] [directly](https://stackoverflow.com/a/48543738) to see if any problems occur.
```

and then we could delete both the useful queries and useful rake tasks sections (as I don't think many of them are very useful in this scenario).

Reply:
Yeah completely agree @kevindew . This page is very noisy; a more focused set of instructions will be far more helpful. I'll update the wording and delete the so-called useful queries and rake tasks.